dvc/project.py

```diff
@@ -388,7 +389,9 @@ def run(self,
         all_stages = self.stages()
 
         self._check_cwd_specified_as_output(cwd, all_stages)
```

Comment:
As already mentioned in https://github.com/iterative/dvc/pull/1460#discussion_r247326575 , let's move cwd check to graph() as well.

Reply:
I've moved this check to the graph